PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -86,21 +98,29 @@ ${CARET7DIR}/wb_command -volume-palette $Jacobian MODE_AUTO_SCALE -interpolate t
 # To detect this situation, and prevent it from occurring, FreeSurferPipeline errors out if the
 # one-step warpfield created here ($OutputOrigT2wToT1w) already exists.]
 convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT1wTransform" --warp1="$dcT1wTransform" --out="$OutputOrigT1wToT1w"
-convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT2wTransform" --warp1="$dcT2wTransform" --postmat="$FinalT2wTransform" --out="$OutputOrigT2wToT1w"
+
+if [ "${T2wPresent}" = "YES" ] ; then
+  convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT2wTransform" --warp1="$dcT2wTransform" --postmat="$FinalT2wTransform" --out="$OutputOrigT2wToT1w"
+fi
```

Comment:
Let's consolidate this into the `if` conditional immediately following.  (i.e., put the `convertwarp` and `applywarp` commands for the T2w image condition back-to-back).

Reply:
Done.